CMakeLists.txt

```diff
@@ -47,14 +47,6 @@ set_property(TARGET SmartHeap::SmartHeap PROPERTY IMPORTED_LOCATION "${CMAKE_SOU
 set_property(TARGET SmartHeap::SmartHeap PROPERTY INTERFACE_INCLUDE_DIRECTORIES "${CMAKE_SOURCE_DIR}/3rdparty/smartheap")
 set_property(TARGET SmartHeap::SmartHeap PROPERTY INTERFACE_COMPILE_OPTIONS "/FI${CMAKE_SOURCE_DIR}/3rdparty/smartheap/SMRTHEAP.HPP")
 
-add_library(mxgeometry STATIC
-  LEGO1/mxgeometry/mxgeometry3d.cpp
-  LEGO1/mxgeometry/mxmatrix.cpp
```

Comment:
Don't forget to add these to somewhere else.

Reply:
These are actually obsolete and served only the purpose of having the `DECOMP_SIZE_ASSERT`s. At some point we should move away from this construct and verify the sizes through the PDB so we don't have this issue with header-only libraries. For now I moved the asserts to an unrelated compilation unit.